### PR TITLE
[DOCS] Add e2e example on cloud native quickstart guide for openvino model server operator

### PR DESCRIPTION
**Pull request by zhlsunshine**
This quick-start guide is an end-to-end example similar with [Quickstart Guide](https://github.com/openvinotoolkit/model_server/blob/main/docs/ovms_quickstart.md). However, it focuses on how to deploy OpenVINO model server as a cloud native `Pod` in Kubernetes cluster, including PV/PVC storage for model, loading model and running inference request in cloud native environment. Moreover, in order to be consistent with original quickstart guide, this guide also adopts [Faster R-CNN with Resnet-50 V1 Object Detection model](https://tfhub.dev/tensorflow/faster_rcnn/resnet50_v1_640x640/1) in TensorFlow format.
Hope this guide can help users build and use OpenVINO more conveniently and quickly in a cloud native environment.

### CODE REVIEW

File: docs/ovms_cloud_native_quickstart.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+openvino-operator.v1.1.0   OpenVINO Toolkit Operator   1.1.0     openvino-operator.v1.0.0   Succeeded
+
+$ kubectl get pods -n operators
+NAME                                                    READY   STATUS    RESTARTS   AGE
+openvino-operator-controller-manager-548888ddc7-jq4mh   2/2     Running   0          5m
+```
+
+### Step 3: Provide a model in PV/PVC
+
+As an end-to-end example, this guide uses `PV/PVC` to store the AI model, and user can also store their model by using [Cloud Storage](using_cloud_storage.md). Before the storage of AI model, please refer to [Model Repository](models_repository.md) to know how to organize components of the model.
```
Comment (pgladkows) [inline docs/ovms_cloud_native_quickstart.md:60]: Please change these links to absolute ones from the model_server repository (https://github.com/openvinotoolkit/model_server/tree/main/docs):

 using_cloud_storage.md
 models_repository.md
 starting_server.md
 model_version_policy.md
 parameters.md
 features.md
 ../demos

File: docs/ovms_cloud_native_quickstart.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+  4 -rw-r--r-- 1 1000 1000     44 Apr 10 09:03 requirements.txt
+```
+
+> **Note**: - Please make sure the `HTTP_PROXY` and `HTTPS_PROXY` are set correctly to install the dependencies libs in `requirements.txt`.
+
+- 4. Review the Results
+
+Showing as previous instruction, User can find output file named `output.jpg` containing inference results. In this case, it should be a modified input image with bounding boxes indicating detected objects and their labels.
+
+![Inference results](cloud_native_quickstart_result.jpeg)
```
Comment (pgladkows) [inline docs/ovms_cloud_native_quickstart.md:293]: Image is missing in this directory

File: docs/ovms_cloud_native_quickstart.md (hunk 1 above)
Comment (zhlsunshine) [reply docs/ovms_cloud_native_quickstart.md]: Thanks @pgladkows for this comment! Done!

File: docs/ovms_cloud_native_quickstart.md (hunk 2 above)
Comment (zhlsunshine) [reply docs/ovms_cloud_native_quickstart.md:293]: I have uploaded the image, done!  Thanks!

**Review by pgladkows — APPROVED** [review]

**Review by dtrawins — APPROVED** [review]

**Pull request merged**